zerver/lib/event_queue.py

```diff
@@ -56,7 +56,7 @@ class ClientDescriptor(object):
     def __init__(self, user_profile_id, user_profile_email, realm_id, event_queue,
                  event_types, client_type_name, apply_markdown=True,
                  all_public_streams=False, lifespan_secs=0, narrow=[]):
-        # type: (int, text_type, int, EventQueue, List[text_type], text_type, bool, bool, int, Iterable[Sequence[text_type]]) -> None
+        # type: (int, text_type, int, EventQueue, List[str], text_type, bool, bool, int, Iterable[Sequence[text_type]]) -> None
```

Comment:
I see that `self.event_types` is never modified once it is set (not just in this function, but also outside it), so maybe `Sequence[str]` would be better here.
Actually there are some places where `self.event_types` is being compared to `None`, like in `ClientDescriptor.accepts_event`. So it should be `Optional[Sequence[str]]`.